src/GraphQL.Tests/Types/ComplexGraphTypeTests.cs

```diff
@@ -96,7 +96,6 @@ public void auto_register_object_graph_type()
             type.Fields.First(f => f.Name == nameof(TestObject.someInt)).Type.ShouldBe(typeof(IntGraphType));
             type.Fields.First(f => f.Name == nameof(TestObject.someNotNullInt)).Type.ShouldBe(typeof(NonNullGraphType<IntGraphType>));
             type.Fields.First(f => f.Name == nameof(TestObject.someBoolean)).DeprecationReason.ShouldBe("Use someInt");
-            type.Fields.First(f => f.Name == nameof(TestObject.someDate)).DefaultValue.ShouldBe(new DateTime(2019, 3, 14));
```

Comment:
This test is invalid; it was testing to see if a default value was set on an output object graph type.  Default values for fields should only exist on input graph types.

Reply:
ok